kolibri/core/assets/src/views/SyncStatusDisplay.vue

```diff
@@ -98,6 +100,10 @@
         message: 'Not connected to server',
         context: 'Status label for a disconnected device.',
       },
+      insufficientStorage: {
+        message: 'Not enough storage',
+        context: 'Status label for a device that does not have enough storage to sync.',
```

Comment:
We usually employ the term `label` to refer to form fields or buttons _labels_, so for translator's clarity, let's change this to `Status indicator for a device...`. 

The same is valid for the context of the string above.

Reply:
Okay, thank you for pointing that out, I will update the context.